src/components/dynamic/user/Dashboard.jsx

```diff
@@ -21,14 +21,10 @@ const Dashboard = () => {
     <div className="h-full font-poppins flex flex-col py-4 gap-3">
       <Title title="Dashboard" />
       <Header email={user.email} name={user.name} />
-      <div className="grid grid-cols-2 h-full overflow-scroll gap-3 justify-center">
-        <div className="h-full">
-          <User user={user} setUser={setUser} edit={edit} setEdit={setEdit} />
-        </div>
+      <div className="grid grid-cols-2 h-full gap-3 justify-center">
```

Comment:
the justify center isn't doing anything cause flex isn't enabled, unless it works on grid